Added AfterPay countries available.

LeasoftKonstantinShapoval opened the pull request.

LeasoftKonstantinShapoval committed:
Added AfterPay countries available.

tiutiunnykleasoft commented on emspay.php:
```diff
@@ -285,6 +285,37 @@ function afterpay_filter_gateways($gateways)
+
+        if (strlen($ap_countries_available) > 0) {
+            $countrylist = explode(",", str_replace(' ', '', $ap_countries_available));
+            if (!in_array(WC()->customer->billing['country'], $countrylist)) {
```
![изображение](https://user-images.githubusercontent.com/60009163/88685027-2f2d7400-d0fe-11ea-9c2b-0b9913686a80.png)

LeasoftKonstantinShapoval replied:
This errors is not for my side. Check server settings, all install requires and follow install order. Checked by new clear WP site.

tiutiunnykleasoft replied:
@LeasoftKonstantinShapoval  : What version wooCommerce you are using ?

tiutiunnykleasoft reviewed (changes requested):
Error on the checkout page

LeasoftKonstantinShapoval committed:
Code Refactoring:
- Add logic of empty val in AfterPay countries validation.
- Update texts and localization for AfterPay countries validation.

tiutiunnykleasoft reviewed (approved):
Ok.

LeasoftKonstantinShapoval committed:
Merge branch 'development' into afterpay_countries
